packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -154,12 +155,28 @@ export class VaFileInput {
   };
 
   private removeFile = () => {
+    this.closeModal();
     this.file = null;
     this.vaChange.emit({ files: [this.file] });
     this.uploadStatus = 'idle';
     this.internalError = null;
   };
 
+  private openModal = () => {
+    // set the status attribute here not in markup or it will have no effect
+    const modal = this.el.shadowRoot.querySelector('va-modal');
+    modal.setAttribute('status', 'warning');
```

Comment:
Does this warning status attribute need to be set manually here, or could it just be permanently set on the `va-modal` component itself?

Reply:
@Andrew565 I believe it must be set like this - if set on the component it does not have any effect (it doesn't get added to the markup on render) - we encountered something like this with va-button components inside other components, where we had to programmatically set the uswds prop. It's some kind of limitation with stencil and nested web-components.